static/app/utils/profiling/colors/utils.tsx

```diff
@@ -238,3 +238,10 @@ export const makeColorMapBySystemVsApplication = (
 
   return colors;
 };
+
+export const colorChannelsToFillStyle = (colorChannels: ColorChannels) => {
```

Comment:
This assumes an 8bit color, which is kind of specific to only the tick rendererer as others still use strings. I would just move it to the top of the renderer for now and not export it through utils. We'll have a chance to create helpers when we move all of the colors off the string definitions

Reply:
dropped it for now